server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala

```diff
@@ -36,5 +36,6 @@ class CreateInteractiveRequest {
   var queue: Option[String] = None
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
+  var scalaVersion: Option[String] = None
```

Comment:
Also for the classpath thing, I don't think we need to worry about it. SparkLaunch and spark-class will handle this, currently what Livy will inject into Spark's classpath is rsc and repl jars.

This field and per-session based Scala version check may not be useful currently, since SPARK_HOME is still a global setting. But as I mentioned this is one task for Livy to support different Sparks.

Also this fix actually will not break current usage and no behavior changes for user.